### PR TITLE
Allow running e2e tests as presubmits on GitHub

### PR DESCRIPTION
**Pull request by uroy-personal**

### CODE REVIEW

**Comment by uroy-personal** [comment]
For the issue : https://github.com/kubernetes/autoscaler/issues/5918

**Comment by uroy-personal** [comment]
Good Morning @jbartosik  , @kgolab , @pbetkier , @mwielgus & @bskiba.

I had to close the PR : https://github.com/kubernetes/test-infra/pull/30893 as my fork got corrupted while rebasing the changes from parent repository.

@pbetkier ,  All the review comments from the previous PR has been incorporated here.  I would like to request your review again on this PR. 
Thanks again for your thorough assistance on this change.

**Comment by uroy-personal** [comment]
I have signed the EasyCLA already. Hope the "EasyCLA — Missing CLA Authorization" check will be cleared soon

**Comment by pbetkier** [comment]
/ok-to-test

Looks fine to me, but let @kwiesmueller handle this as the new owner :)

**Comment by kwiesmueller** [comment]
Or rather use a different dashboard name?

**Comment by uroy-personal** [comment]
Good Morning @kwiesmueller ,
Yes his dashboard name has been used in the sig-autoscaling-config.yaml ( line no 139 ).  Let me change the name. What name should we use? 
**testgrid-tab-name: autoscaling-vpa-full-presubmits**?

**Comment by kwiesmueller** [comment]
That name sounds good, yes.

**Comment by umesh-roy-sp** [comment]
Hi @kwiesmueller ,
I made the changes and trigger retest. The **pull-test-infra-verify-lint**  test is failing with the message:

config/prow/cluster/build/build_kubernetes-external-secrets_deployment.yaml
  39:11     warning  comment not indented like content  (comments-indentation)
ERROR: yamllint failed

I have not been able to identity the issue so far. When I validate the whole file using a yamllint, I get no error.  I will look into the issue later today.

**Review by jbartosik — CHANGES_REQUESTED** [review]
Please make sure that the CLA presubmit is passing, without it we can't accept the contribution.

**Comment by uroy-personal** [comment]
Hi @jbartosik , @kwiesmueller 
I got the EasyCLA step passed. Please approve.  Thanks.

**Comment by uroy-personal** [comment]
Good afternoon @MaciekPytel , @smg247 ,
Request your review on this PR please.

File: config/testgrids/openshift/redhat-openshift-ocp-release-4.13-informing.yaml [hunk 1]
```diff
@@ -540,6 +540,33 @@ dashboards:
     results_url_template:
       url: https://prow.ci.openshift.org/job-history/<gcs_prefix>
     test_group_name: periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-upgrade-aws-ovn-arm64
+  - base_options: width=10&exclude-filter-by-regex=Monitor%5Cscluster&exclude-filter-by-regex=%5Eoperator.Run%20template.*container%20test%24
```
Comment (laoj2) [inline config/testgrids/openshift/redhat-openshift-ocp-release-4.13-informing.yaml:543]: Can you move the config/testgrids/openshift/... changes to another PR? Those appear to be unrelated to the VPA presubmit tests

Comment (uroy-personal) [reply config/testgrids/openshift/redhat-openshift-ocp-release-4.13-informing.yaml:543]: Hi @laoj2 ,
I did not add or make any changes to config/testgrids/openshift/. When I raised the PR initially I could only see my changes in a single file. Now I see 3 files got changed as part of this PR. 

No idea why config/testgrids/openshift/ changes are pulled in into this PR..

**Comment by uroy-personal** [comment]
Good morning @laoj2 ,
While rebasing with the origin, my repository got so many conflicts for some reasons. It was taking too much time to fix for this change. So I deleted my fork and recreated it. This pull request is closed. 
I have raised a new one: https://github.com/kubernetes/test-infra/pull/30994

Could you please review it. 
Thanks.
